ruby-challenges.rb

```diff
@@ -33,3 +60,37 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+class Super_Bike
+  attr_accessor :model, :wheels, :current_speed
+
+  def initialize(model_param, wheels_param = 2, current_speed_param = 0)
+    @model = model_param
+    @wheels = wheels_param
+    @current_speed = current_speed_param
+  end
+
+  def bike_info
+    "The #{@model} has #{@wheels} wheels and is going #{@current_speed} mph."
+  end
+
+  def pedal_faster(pedal_param)
+    @current_speed += pedal_param
+  end
+
+  def brake(brake_param)
+    @current_speed -= brake_param
+    if @current_speed <= 0
+      @current_speed = 0
+    end
+    @current_speed
+  end
+end
+p '------------------3b------------------'
+p my_bike = Super_Bike.new("BatMan Bike")
+p my_bike.pedal_faster(10)
+p my_bike.pedal_faster(18)
+p my_bike.brake(5)
+# p my_bike.brake(6)
+# p my_bike.brake(5)
+p my_bike.brake(25)
```

Comment:
Nice job Jason. In this prompt you can combine these two classes into a single Bike class.